<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevicesWidget</class>
 <widget class="QWidget" name="DevicesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="p_buttonCheckAlive">
     <property name="text">
      <string>Check Alive</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="p_buttonUpdateNodesData">
     <property name="text">
      <string>Update nodes data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="p_buttonExpandAll">
     <property name="text">
      <string>Expand all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTreeView" name="p_treeViewAliveNodes">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="p_buttonUpdateData">
     <property name="text">
      <string>Update all devices data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="p_buttonCollapseAll">
     <property name="text">
      <string>Collapse all</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="p_buttonChangeReadingPeriod">
     <property name="text">
      <string>Change reading period</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
